src/constants.js

```diff
@@ -0,0 +1,11 @@
+import L from "leaflet";
```

Comment:
`src/constants.js` could be placed inside the `/utils` folder.

Reply:
Agreed. Also, avoid single-letter names. This is something I see used a lot, and it's something I wish people would do less because it impairs code readability. Variable names should be descriptive enough to make their function or purpose clear.